tests/test_ipa_trust.py

```diff
@@ -1287,3 +1288,44 @@ def test_agent_without_package(self):
         assert result.source == 'ipahealthcheck.ipa.trust'
         assert result.check == 'IPATrustPackageCheck'
         sys.modules['ipaserver.install'] = save
+
+
+class TestConfiguration(BaseTest):
+
+    def test_ipakrbauthzdata_positive(self):
+        framework = object()
+        registry.initialize(framework, config.Config)
+        f = IPAauthzdatapacCheck(registry)
+
+        m_api.Command.config_show.side_effect = [{
+            'result': {
+                'ipakrbauthzdata': ['MS-PAC', 'nfs:NONE',]
```

Comment:
Running lint on my local machine wants a space after the trailing comma:

`'ipakrbauthzdata': ['MS-PAC', 'nfs:NONE', ]`

I don't know why the github CI didn't pick it up. Diff version of python perhaps.